fem/bilininteg.cpp

```diff
@@ -626,7 +626,7 @@ void DiffusionIntegrator::AssembleElementVector(
 
 void DiffusionIntegrator::ComputeElementFlux
 ( const FiniteElement &el, ElementTransformation &Trans,
-  Vector &u, const FiniteElement &fluxelem, Vector &flux, int with_coef )
+  Vector &u, const FiniteElement &fluxelem, Vector &flux, bool with_coef )
```

Comment:
Thanks for this change! Using ints instead of bools is one of my personal pet peeves.